src/settings.rs

```diff
@@ -101,6 +106,8 @@ static DEFAULT_EXCLUDE: Lazy<Vec<FilePattern>> = Lazy::new(|| {
     ]
 });
 
+static DEFAULT_DUMMY_VARIABLE_RGX: Lazy<Regex> = Lazy::new(|| Regex::new("^_+$").unwrap());
```

Comment:
pylint uses the following default value:

```
_+$|(_[a-zA-Z0-9_]*[a-zA-Z0-9]+?$)|dummy|^ignored_|^unused_
```

https://pylint.pycqa.org/en/latest/user_guide/configuration/all-options.html#dummy-variables-rgx